### PR TITLE
Implement previous context marks (add to jump list on m' / m`)

### PR DESCRIPTION
**Pull request by jose-elias-alvarez**
**What this PR does / why we need it**:
Implements previous context marks as described in Vim's `help m'` / `` help m` ``.

**Which issue(s) this PR fixes**
https://github.com/VSCodeVim/Vim/issues/4873

**Special notes for your reviewer**:
First contribution to the project, so please let me know if there's anything I can do differently. Thanks!

### CODE REVIEW

File: src/jumps/jumpTracker.ts [hunk 1]
```diff
@@ -69,8 +69,8 @@ export class JumpTracker {
    * @param from - File/position jumped from
    * @param to - File/position jumped to
    */
-  public recordJump(from: Jump, to: Jump) {
-    if (from.isSamePosition(to)) {
+  public recordJump(from: Jump, to: Jump | null) {
```
Comment (J-Fields) [inline src/jumps/jumpTracker.ts:72]: Use `undefined` instead of `null` please

File: test/historyTracker.test.ts [hunk 2]
```diff
@@ -42,6 +44,27 @@ suite('historyTracker unit tests', () => {
       historyTracker = setupHistoryTracker();
     });
 
+    test('can set previous context mark from single quote', () => {
```
Comment (J-Fields) [inline test/historyTracker.test.ts:47]: Could you also add a test in `jumpTracker.test.ts`?

**Review by J-Fields — CHANGES_REQUESTED** [review]
Thanks a bunch! LGTM other than those few nits

**Comment by jose-elias-alvarez** [comment]
Thanks for the feedback! I changed the type and added a test.

**Comment by J-Fields** [comment]
Thanks!

**Pull request merged**